### ISSUE TITLE
goose3 skips content from HTML lists (ul, ol)

### ISSUE THREAD

**Issue by polosatyi** [goose3/goose3#48]
Hey, guys!

It seems like **goose3** can not extract content from html lists (`ul`, `ol` tags). For example, only "Some text A. Some text C." will be extracted from:
```
<p>Some text A.</p><ul><li>Some text B.</li></ul><p>Some text C.</p>
```
Is it normal behavior? Is it possible to fix?

**Comment by polosatyi** [comment]
Or could you point me where can I modify this behavior, please?

**Comment by polosatyi** [comment]
Example: https://plantcaretoday.com/how-to-grow-and-care-for-bougainvillea.html

**Comment by barrust** [comment]
Sadly, this is not a feature. I think we should add this but I haven't been able to look into it. 

I think the location that is likely the culprit is the `goose3/extractors/content.py` There seems to be a place where we are not converting the list (and headers #17) to paragraphs.

**Comment by polosatyi** [comment]
My issue was solved when I changed this line https://github.com/goose3/goose3/blob/master/goose3/extractors/content.py#L353 to
```
if e_tag not in ('p', 'ul', 'ol', ):
```
Can I create a pull request?

**Comment by barrust** [comment]
@polosatyi I was thinking about this PR a bit more and I wanted to add a bit of formatting to the returned list as I think it will make it easier to read and follow in the cleaned up text. I will submit a PR for review, if you wouldn't mind looking at it that would be great!

**Comment by polosatyi** [comment]
Honestly, pretty lists is not so important for me. More than that maybe I will have to delete them in my work.

**Comment by barrust** [comment]
I should be able to add a parameter to optionally add them or take them out. Let me know if that would be helpful!

**Comment by barrust** [comment]
resolved by #51

**Comment by polosatyi** [comment]
Thank you!
